SPECS/tpm2-tools/tpm2-tools.spec

```diff
@@ -23,23 +30,34 @@ The source repository for the TPM (Trusted Platform Module) 2 tools
 %autosetup -p1
 
 %build
-%configure \
-    --disable-static
-
-make %{?_smp_mflags}
+sed -i "/compatibility/a extern int BN_bn2binpad(const BIGNUM *a, unsigned char *to, int tolen);" lib/tpm2_openssl.c
```

Comment:
I know this is direct from Photon, but this looks pretty horrible. What this is doing is *literally* the entire point of patch files.

Reply:
lol.  agree entirely.